packages/dashboard-frontend/src/services/workspace-client/devworkspace/devWorkspaceClient.ts

```diff
@@ -205,6 +226,7 @@ export class DevWorkspaceClient extends WorkspaceClient {
     });
     const cheTheiaPluginsContent = optionalFilesContent['.che/che-theia-plugins.yaml'];
     const vscodeExtensionsJsonContent = optionalFilesContent['.vscode/extensions.json'];
+
     const cheTheiaPluginsDevfileResolver = container.get(CheTheiaPluginsDevfileResolver);
```

Comment:
well, do I understand correctly that we invoke theia logic for all editors, including non-theia?

Reply:
yes I'll provide fixup PR when development mode will run smoothly
(but anyway the lib won't do anything if it's not theia, we're just loosing precious time)

Reply:
>  we're just loosing precious time

+ making dashboard logic impossible to understand )

Reply:
yes I agree. I would jut not mix stuff altogether.
There'll be probably a new library for enhancing devfiles for other editors as well.